### ISSUE TITLE
Header elements are not accessible

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/education.wsu.edu#3]
It looks like the color used for most (all?) header elements has a contrast ratio of 3.94:1.
